assessments-service/src/test/java/com/transformuk/hee/tis/assessment/service/api/AssessmentResourceTest.java

```diff
@@ -493,6 +493,20 @@ public void deleteAssessmentShouldReturnOK() throws Exception {
         .andExpect(status().isOk());
   }
 
+  @Test
+  public void deleteAssessmentShouldReturnBadRequestWhenAssessmentForTraineeNotFound() throws Exception {
+    when(assessmentServiceMock.deleteAssessment(ASSESSMENT_ID)).thenReturn(false);
+    mockMvc.perform(delete("/api/trainee/assessments/{assessmentId}", ASSESSMENT_ID))
+        .andExpect(status().isBadRequest());
+  }
+
+//  @Test
+//  public void deleteAssessmentShouldReturnError() throws Exception {
+//    when(assessmentServiceMock.deleteAssessment(ASSESSMENT_ID)).thenReturn(true);
+//    mockMvc.perform(delete("/api/trainee/assessments/{assessmentId}", ASSESSMENT_ID))
+//        .andExpect(status().);
+//  }
+
```

Comment:
The comments above need to be removed.

Reply:
ahh completely forgot that I had commented that out thanks!